docs/source/tutorial.rst

```diff
@@ -0,0 +1,126 @@
+Tutorial
+--------
+
+For this tutorial we are going to use a well known case file from `NIST <https://cfreds.nist.gov/all/NIST/HackingCase>`_.
+In this tutorial we assume Linux is used, on other operating systems most steps are the same but details may differ.
+First of all, we install Python and create a venv:
+
+.. code-block:: console
+
+    $ python3 -m venv dissect
+
+Now activate the virtual environment to get some work done:
+
+
+.. code-block:: console
+
+    $ source dissect/bin/activate
+    
+
+Let's download the case files. At the time of writing they are hosted at nist.gov.
+On Linux:
+
+.. code-block:: console
+
+    $ for i in {1..8}; do curl https://cfreds-archive.nist.gov/images/hacking-dd/SCHARDT.00$i -o SCHARDT.00$i; done
+    
+.. note ::
+
+    You can also download them manually of course.
+    
+Now, we are going to do some basic operations on this image. If you like, you can merge them together first,
+although this is not strictly necessary:
+
+.. code-block:: console
+
+    $ for i in `ls SCHARDT.00*`; do cat $i >> SCHARDT.img; done
```

Comment:
This is in my opinion only making things quite difficult for an unexperienced forensic investigator. For someone experienced with bash this is quite logical, however I might guess most people will just plainly copy paste the command. This is an Dissect tutorial, not a bash one-liner showcase.

Reply:
People tend to just copy-paste these lines. They don't have to understand it fully. This is done in many tutorials to help inexperienced users to get to a workable begin state.

Reply:
ok